heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is typically used to derive cluster status based
+    # on the up or down status of its members.
```

Comment:
Yes, indeed. I am currently looking at the status_of_members,  trying to understand when it can be useful…